wp-content/themes/core/assets/js/editor/ready.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @module
+ * @exports ready
+ * @description The core dispatcher for the dom ready event javascript.
+ */
+
+/**
+ * @function init
+ * @description The core dispatcher for init across the codebase.
+ */
+
+const init = () => {
+	// intentionally left blank for now
+
+	console.info(
+		'Moose Editor: Initialized all javascript that targeted document ready.'
```

Comment:
Using `Moose` here doesn't provide any additional value and could be confusing to clients.  Moose is more of an internal thing so we know what we are talking about. We can just have `Editor: ....`